drivers/include/saul/periph.h

```diff
@@ -32,13 +32,20 @@ extern "C" {
 #endif
 
 #ifdef MODULE_SAUL_GPIO
+typedef enum {
+    SAUL_GPIO_INVERTED   = 1 << 0, /**< pin is used as inverted */
+    SAUL_GPIO_INIT_CLEAR = 1 << 1, /**< set pin inactive after initialization */
+    SAUL_GPIO_INIT_SET   = 1 << 2, /**< set pin active after initialization */
```

Comment:
here and in other places of this PR: I would tend to remove the comma after the last element

Reply:
Forget about it. As we don't use C89 there is no reason, so keep it as is.